staging/provisioning/windows/kubeproxystart.ps1

```diff
@@ -44,5 +44,5 @@ if (Test-Path "$global:KubeDir\runprocess.dll") {
     Add-Type -Path "$global:KubeDir\run-process.cs"
 }
 $exe = "$global:KubeDir\kube-proxy.exe"
-$args = ($global:KubeproxyArgList -join " ")
+$args = $global:KubeproxyArgList -join " "
```

Comment:
is this change clean up or did it have an effect on the outcome?

Reply:
From my test, both works. But if you execute `$args = ($global:KubeproxyArgList -join " ")` in PowerShell and then execute `echo $args`, you will not get any output.